src/app/login/login.service.ts

```diff
@@ -8,7 +8,7 @@ import { LoginItem } from './login-item';
 @Injectable()
 export class LoginService {
   // private githubUrl = 'app/loginStatus';
-  private githubUrl = 'http://localhost:8081/api/login/authorize ';  // URL to web api
+  private githubUrl = 'http://localhost:8080/api/login/authorize ';  // URL to web api
 
```

Comment:
Why did you change this? It is set to 8081 so that it doesn't conflict with the backend when running on localhost.